modules/core/src/lib/attribute-transition-manager.js

```diff
@@ -27,7 +27,7 @@ export default class AttributeTransitionManager {
 
     if (Transform.isSupported(gl)) {
       this.isSupported = true;
-    } else {
+    } else if (gl) {
```

Comment:
Maybe add a short comment reminding the maintainer that this class must be instantiatable without a gl context?